videolibrary_list.go

```diff
@@ -0,0 +1,21 @@
+package bunny
+
+import "context"
+
+// VideoLibraries represents the response of the List Video Library API endpoint.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/videolibrarypublic_index
+type VideoLibraries PaginationReply[VideoLibrary]
+
+// List retrieves the Video Libraries.
+// If opts is nil, DefaultPaginationPerPage and DefaultPaginationPage will be used.
+// if opts.Page or or opts.PerPage is < 1, the related DefaultPagination values are used.
+//
+// Bunny.net API docs: https://docs.bunny.net/reference/videolibrarypublic_index
+// TODO: add `includeAccessKey` path param
```

Comment:
Here's the function that needs to pass the `includeAccessKey` option.

Reply:
@fho I believe I implemented this the way you were wanting. Let me know if you need me to squash or make more changes.